cmd/crowdsec-cli/dashboard.go

```diff
@@ -304,3 +323,14 @@ func passwordIsValid(password string) bool {
 	return true
 
 }
+
+func checkSystemMemory() (bool, error) {
+	totMem := memory.TotalMemory()
+	if totMem == 0 {
+		return true, errors.New("Unable to get system total memory")
+	}
+	if uint64(math.Pow(2, 30)) >= totMem {
```

Comment:
Don't know math.Pow is needed here as the value will always resolve to ```1073741824``` so we could swap this out and hard code the value